lib/bulk_data_test_kit/v2.0.0/client/postman/delete.postman_collection.json

```diff
@@ -0,0 +1,117 @@
+{
+	"info": {
+		"name": "Bulk Data Client - System Export Delete",
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
+	},
+	"item": [
+		{
+			"name": "Kickoff",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"exec": [
+							"pm.test(\"kick-off was accepted\", function () {",
+							"  pm.response.to.have.status(202);",
+							"});",
+							"",
+							"pm.collectionVariables.set(\"status-url\", pm.response.headers.get('Content-Location'));",
+							""
+						],
+						"type": "text/javascript",
+						"packages": {}
+					}
+				}
+			],
+			"request": {
+				"method": "GET",
+				"header": [],
+				"url": {
+					"raw": "{{kickoff-base-url}}/$export",
+					"host": [
+						"{{kickoff-base-url}}"
+					],
+					"path": [
+						"$export"
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "Delete",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"exec": [
+							"let responseData = pm.response.json();",
```

Comment:
This `let responseData = pm.response.json();` is causing Postman to throw a JSON Error: `JSONError: No data, empty input at 1:1` since there is no body returned from the delete request. I think you can just remove that line from your script.

Reply:
Oops! Good catch, I've removed it.